components/rootless_studio/tests/bats/secrets.bats

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bats
+
+# TODO (CM): Better parameterize the HAB_DOCKER_STUDIO_IMAGE
+# environment variable.
+#
+# As it is, you can just execute these tests with that in the
+# environment, but it would be nice to make it explicit.
+#
+# HAB_DOCKER_STUDIO_IMAGE=my_image bats secrets.bats
+
+@test "hab pkg build (rootless Docker image): secrets are passed" {
```

Comment:
Since we are leveraging the hab cli to run these tests, we should probably pull them out of the rootless studio and into the general testing directory for the cli

Reply:
Yeah, once we get these plugged into automation, the organization will definitely change. This was just to get some tests in place for the regressions being fixed.